backend/lib/services/shoots.js

```diff
@@ -227,27 +227,31 @@ exports.info = async function ({ user, namespace, name }) {
 
       const seedShootNS = _.get(shoot, 'status.technicalID')
       if (!_.isEmpty(seedShootNS)) {
-        const monitoringSecret = await kubernetes.core(kubernetes.fromKubeconfig(seedKubeconfig)).ns(seedShootNS).secrets.get({ name: 'monitoring-ingress-credentials' })
-          .catch(err => {
-            if (err.code === 404) {
-              return
-            }
-            logger.error('failed to fetch monitoring secret: %s', err)
-            throw err
-          })
-        if (monitoringSecret) {
-          _
-            .chain(monitoringSecret)
-            .get('data')
-            .pick('username', 'password')
-            .forEach((value, key) => {
-              if (key === 'password') {
-                data['monitoring_password'] = decodeBase64(value)
-              } else if (key === 'username') {
-                data['monitoring_username'] = decodeBase64(value)
+        const components = ['monitoring', 'logging']
+        for (let i = 0; i < components.length; i++) {
+          const component = components[i]
+          const componentSecret = await kubernetes.core(kubernetes.fromKubeconfig(seedKubeconfig)).ns(seedShootNS).secrets.get({ name: `${component}-ingress-credentials` })
```

Comment:
in addition, instead of fetching the secrets sequentially, use Promise.all
you could also consider to move parts of this method to an own function for better re-usability and (just as an example) pass the kubernetes client and components for which secrets should be fetched and the data object